README.md

```diff
@@ -101,7 +101,21 @@ We'll walk through the steps of becoming a transcoder on the test network.  To l
 
 - Wait for the next round to start, and your transcoder will become active.
 
-- If running on Rinkeby or mainnet, ensure your transcoder is *publicly accessible* in order to receive jobs from broadcasters. The only port that is required to be public is the one that was set during the transcoder registration step (default 8935).
+- If running on Rinkeby or mainnet, ensure your orchestrator is *publicly accessible* in order to receive jobs from broadcasters. The only port that is required to be public is the one that was set during the transcoder registration step (default 8935).
+
+### Standalone Orchestrators
+
+Orchestrators can be run in standalone mode without an attached transcoder. Standalone transcoders will need to connect to this orchestrator in order for the orchestrator to process jobs.
+
+- `livepeer --rinkeby --orchestrator -orchSecret asdf`
+
+The orchSecret is a shared secret used to authenticate remote transcoders.
```

Comment:
Looks like secret can be any arbitrary string?  If so, let's put it in the comment.